src/routes/docs/tutorials/sveltekit-ssr-auth/step-3/+page.markdoc

```diff
@@ -27,7 +27,6 @@ export const SESSION_COOKIE = 'my-custom-session';
 
 export function createAdminClient() {
     const client = new Client()
-        .setEndpoint(PUBLIC_APPWRITE_ENDPOINT)
```

Comment:
Other parts of the tutorial needs to be updated too since this is removed, right?